usabilla-react-native/README.md

```diff
@@ -79,9 +84,23 @@ This method accepts as parameter a valid JSON object with two limitations:
 
 **NOTE**: Feedback sent without respecting those limitations will still be received and saved, but the values of the customer input context won't be displayed nor exported.
 
+### Campaigns
+In order to be able to run campaigns in your app, you should first, start by initializing the SDK:
+
+`usabilla.initialize("YOUR_APP_ID")`
+
+This call loads and updates all your campaigns locally and you can start targeting them by seding events from your app:
+
+`usabilla.sendEvent("YOUR_EVENT_NAME")`
+
+The Usabilla SDK allows you to reset all the campaign data:
```

Comment:
Also here to keep consistency with other features we should say something like `with the method` or `calling:` or other